.github/workflows/create-release-pr.yml

```diff
@@ -31,7 +31,9 @@ jobs:
         run: echo "Bumping to version ${{ steps.cz.outputs.version }}"
 
       - name: Update Cargo.lock
-        run: cargo build -p ic-response-verification
+        run: |
+          cargo build
+          pnpm build
```

Comment:
Not all `Cargo.lock` files were being updated and they would end up being updated by subsequent PRs. Unfortunately we need to build everything for the lock files to be updated.